FutureNorthants/northants-design-system
# Bump npm dependencies to latest versions

Update npm dependencies. 

## To test
- checkout this branch and `npm install`
- Run `npm outdated` to ensure nothing is outdated
- Run `npm run build` to test the build runs ok
- Run `npm run test` to run tests
- Run `npm run dev` to launch storybook and manually test pages
- Run `yalc publish` then, in the frontend, run `yalc add northants-design-system && yarn install`. 
- Run `yarn build && yarn start` and test the frontend builds and works ok
